### PR TITLE
Reports: Add region total rows and copy/styling updates

### PR DESCRIPTION
**Pull request by claudiovallejo**
**Story card:** [ch1399](https://app.clubhouse.io/simpledotorg/story/1399/add-all-rows-to-facility-comparison-tables)

## Because

1. It makes it easier to see (and verify) accuracy of facility data
2. It shows the region's name as the "All" header (per CVHOs request)

## This addresses

**Add total rows to facility comparison tables**
image

**Replace "All" with region's name in District Report tables**
image

**Replace "All" with region's name in Facility Report tables**
image

**Update cohort chart key label and tooltip copy**
image
image

**Format cohort chart tooltips to properly display "patient" in singular/plural**
image

### CODE REVIEW

File: app/views/reports/regions/cohort.html.erb [hunk 1]
```diff
@@ -42,7 +42,7 @@
                     data-toggle="tooltip"
                     data-placement="top"
                     data-trigger="hover focus click"
-                    title="<%= number_with_delimiter(cohort["no_bp"]) %> patients didn't have a BP taken"
+                    title="<%= number_with_delimiter(cohort["no_bp"]) %> <% if cohort["no_bp"] == 1 %>patient<% else %>patients<% end %> didn't have a BP taken"
```
Comment (rsanheim) [inline app/views/reports/regions/cohort.html.erb:45]: [pluralize](https://apidock.com/rails/ActionView/Helpers/TextHelper/pluralize) is your friend here (and below) .  No need to do this by hand. 😁

**Review by rsanheim — APPROVED** [review]
One place where you can use a Rails view helper, otherwise LGTM.  approving to unblock.

File: app/views/reports/regions/cohort.html.erb (hunk 1 above)
Comment (harimohanraj89) [reply app/views/reports/regions/cohort.html.erb:45]: Thanks Rails :heart:

Comment (claudiovallejo) [reply app/views/reports/regions/cohort.html.erb:45]: Pluralize is so nice ❤️

**Pull request merged**